pennylane/math/utils.py

```diff
@@ -269,7 +269,13 @@ def _get_interface_of_single_tensor(tensor):
     """
     namespace = tensor.__class__.__module__.split(".")[0]
 
-    if namespace in ("pennylane", "autograd"):
+    if namespace == "pennylane":
+        # pylint: disable=import-outside-toplevel
+        from pennylane.capture import enabled
+
+        return "jax" if enabled() else "autograd"
+
+    if namespace == "autograd":
         return "autograd"

```

Comment:
Unfortunately I've had to resort to importing pennylane here. I initially tried just using `ar.infer_backend` if `namespace == "pennylane"`, but when using `qml.numpy`, the namespace is `"pennylane"`, but `ar.infer_backend` returns `numpy` because the first thing that that function checks is `issubclass(type(tensor), numpy.ndarray)`

Reply:
Resolved.